### PR TITLE
Update the changelog to reflect recent PRs

### PR DESCRIPTION
**Pull request by gouthamve**
Making my life as a release shepherd easy.

Signed-off-by: Goutham Veeramachaneni <gouthamve@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 ## master / unreleased
 
-* [FEATURE] You can now specify `http_config` on alert receivers #929
 * [CHANGE] Flags changed due to changes upstream in Prometheus Alertmanager #929:
   * `alertmanager.mesh.listen-address` is now `cluster.listen-address`
   * `alertmanager.mesh.peer.host` and `alertmanager.mesh.peer.service` can be replaced by `cluster.peer`
   * `alertmanager.mesh.hardware-address`, `alertmanager.mesh.nickname`, `alertmanager.mesh.password`, and `alertmanager.mesh.peer.refresh-interval` all disappear.
 * [CHANGE] Retention period should now be a multiple of periodic table duration #1564
+* [CHANGE] The name label for the chunks memcache in all `cortex_cache_` is now `chunksmemcache` (before it was `memcache`) #1569
+* [FEATURE] Honor https for custom S3 URLs #1603
```
Comment (gouthamve) [inline CHANGELOG.md]: Though this can be considered a Breaking change too :)

Obligatory xkcd https://xkcd.com/1172/

Comment (tomwilkie) [reply CHANGELOG.md]: We can do breaking changes in 0.x releases.

Comment (tomwilkie) [reply CHANGELOG.md]: s/https/HTTPS/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 ## master / unreleased
 
-* [FEATURE] You can now specify `http_config` on alert receivers #929
 * [CHANGE] Flags changed due to changes upstream in Prometheus Alertmanager #929:
   * `alertmanager.mesh.listen-address` is now `cluster.listen-address`
   * `alertmanager.mesh.peer.host` and `alertmanager.mesh.peer.service` can be replaced by `cluster.peer`
   * `alertmanager.mesh.hardware-address`, `alertmanager.mesh.nickname`, `alertmanager.mesh.password`, and `alertmanager.mesh.peer.refresh-interval` all disappear.
 * [CHANGE] Retention period should now be a multiple of periodic table duration #1564
+* [CHANGE] The name label for the chunks memcache in all `cortex_cache_` is now `chunksmemcache` (before it was `memcache`) #1569
```
Comment (tomwilkie) [inline CHANGELOG.md]:
```suggestion
* [CHANGE] The value for the name label for the chunks memcache in all `cortex_cache_` metrics is now `chunksmemcache` (before it was `memcache`) #1569
```

**Comment by csmarchbanks** [comment]
Perhaps add something about ingester.spread-flushes being consistent between ingesters? https://github.com/cortexproject/cortex/pull/1578

**Pull request merged**